lms/templates/courseware/course_updates.html

```diff
@@ -4,26 +4,47 @@
   % for index, update in enumerate(visible_updates):
     <article class="updates-article">
       % if not update.get("is_error"):
-        <h2 class="date">${update.get("date")}</h2>
-        <a class="toggle-visibility-button" data-hide="${_('Hide')}" data-show="${_('Show')}"></a>
+        <h2 class="date" id="msg-date-${index}">${update.get("date")}</h2>
+        <button
+            class="toggle-visibility-button"
+            data-hide="${_('Hide')}"
+            data-show="${_('Show')}"
+            aria-describedby="msg-date-${index}"
+            aria-controls="msg-content-${index}"
+            aria-expanded="true"
+        ></button>
       % endif
-      <div class="toggle-visibility-element article-content ${'hidden' if index >= 1 else ''}">
+      <div class="toggle-visibility-element article-content ${'hidden' if index >= 1 else ''}" id="msg-content-${index}">
         ${update.get("content")}
       </div>
     </article>
   % endfor
 </div>
-<div class="old-updates hidden toggle-visibility-element">
-  % for update in hidden_updates:
+
+% if len(hidden_updates) > 0:
+<button
+    class="toggle-visibility-button show-older-updates"
+    data-hide=""
+    data-show="${_('Show Earlier Course Updates')}"
+    aria-expanded="false"
+    aria-controls="old-updates"
+></button>
+% endif
+
+<div class="old-updates hidden toggle-visibility-element" id="old-updates">
+  % for index, update in enumerate(hidden_updates):
     <article class="updates-article">
-      <h2 class="date">${update.get("date")}</h2>
-      <a class="toggle-visibility-button" data-hide="${_('Hide')}" data-show="${_('Show')}"></a>
-      <div class="toggle-visibility-element article-content hidden">${update.get("content")}</div>
+      <h2 class="date" id="msg-date-${index + len(visible_updates)}">${update.get("date")}</h2>
+      <button
+          class="toggle-visibility-button"
+          data-hide="${_('Hide')}"
+          data-show="${_('Show')}"
+          aria-describedby="msg-date-${index + len(visible_updates)}"
+          aria-controls="msg-content-${index + len(visible_updates)}"
+          aria-expanded="false"
+      ></button>
```

Comment:
@peter-fogg does this ```<button>``` get populated with a text value at some point?  Not sure if you are using the data-hide and data-show values to dynamically populate the text node, but it needs to have some sort of text value that clearly indicates what the control does.  I think that is what is going on here, but I just wanted to make sure.